Assets/MixedRealityToolkit.Tests/PlayModeTests/Utilities/AssertHelpers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    /// <summary>
+    /// Test helpers that help with assertions of non-trivial object types.
+    /// </summary>
+    public class AssertHelpers
+    {
+        /// <summary>
+        /// Validates that the given Vector3s are almost equal, in particular that the corresponding
+        /// X, Y, and Z values of the two vectors differ by at most the specified tolerance (which
+        /// defaults to .02)
+        /// </summary>
+        /// <remarks>
+        /// Note that the tolerance is not based on the total summed delta - if any of the individual
+        /// parts exceed the tolerance this assert will fail the test.
+        /// </remarks>
+        public static void AlmostEquals(Vector3 expected, Vector3 actual, float tolerance = 0.02f)
```

Comment:
Could use [`TestUtilities.AssertAboutEqual`](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ac2b8108f85aaef32b7f1d6022c1af1d19b154b2/Assets/MixedRealityToolkit.Tests/TestUtilities.cs#L192)? Unless the distinction between the vectors differing by their components vs by distance is important for these tests? In either case, it may be nice to move `TestUtilities.AssertAboutEqual` into this new `AssertHelpers` class.